src/ui/src/abstractlogview.cpp

```diff
@@ -2420,12 +2415,12 @@ void AbstractLogView::drawTextArea( QPaintDevice* paintDevice )
             const auto matchPart = QStringView{ logLine }.mid( match.startColumn(), match.size() );
 #endif
             const auto expandedMatchLength
-                = untabify( matchPart.toString(), expandedPrefixLength ).size();
+                = untabify( matchPart.toString(), static_cast<int>(expandedPrefixLength) ).size();
```

Comment:
The same here, should be LineLength::UnderlyingType.

Reply:
Should we change the second parameter `int initialPosition ` to `LineLength::UnderlyingType` in `inline QString untabify( QString&& line, int initialPosition = 0 )`?

Reply:
These type changes might explode to changing many places in code. I'm in the middle of types cleanup and I'd prefer this PR to be as compact as possible.